infrastructure/variables.tf

```diff
@@ -56,6 +56,22 @@ variable "gov_pay_url" {
   default = "https://publicapi.payments.service.gov.uk/v1/payments"
 }
 
+variable "liberata_oauth2_base_url" {
+  default = "https://bpacustomerportal.liberata.com/pba/public/api/v2"
+}
+
+variable "liberata_api_account_url" {
+  default = "https://bpacustomerportal.liberata.com/pba/public/api/v2/account"
+}
+
+variable "liberata_oauth2_authorize_url" {
```

Comment:
Can't we just externalize base url only here and rest of the paths in the application itself?

Reply:
what is the advantage of doing that instead of having it in the config, but not the code itself? What if Liberata changes the urls? Isn't it easier to have it in the config then and just update it and restart the system?

Reply:
Well, change of endpoint subpath means a new version of API - so rarely change. Another option is constructing them in main.tf based on base URL.